TelerikBlazor.sln

```diff
@@ -1,31 +1,43 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
-VisualStudioVersion = 16.0.28531.58
+VisualStudioVersion = 16.0.28711.60
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TelerikBlazor.Server", "TelerikBlazor\TelerikBlazor.Server\TelerikBlazor.Server.csproj", "{DB13D0B5-39C3-455E-96DF-D34B342290BA}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TelerikBlazor.Server", "TelerikBlazor.Server\TelerikBlazor.Server.csproj", "{DDFB771E-5733-40F3-B9D4-844501100BBC}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TelerikBlazor.App", "TelerikBlazor\TelerikBlazor.App\TelerikBlazor.App.csproj", "{FA4EE663-93A7-4BFA-B92E-58406DEE8818}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TelerikBlazor.Client", "TelerikBlazor.Client\TelerikBlazor.Client.csproj", "{F1B213CF-C781-4C1A-8692-63D8E978AD3E}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TelerikBlazor.Shared", "TelerikBlazor.Shared\TelerikBlazor.Shared.csproj", "{7A77AA19-B538-491A-B701-76F9A1E5A02A}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Kendo.Blazor", "D:\work\git\kendo-blazor\Kendo.Blazor\Kendo.Blazor.csproj", "{AAE47115-963A-4A97-BB44-E12441F5CFE9}"

```

Comment:
This reference causes the wrong loading when opening the solution. It should be removed.